includes/admin/class-bc-templates.php

```diff
@@ -302,7 +302,7 @@ public function add_templates() {
 		<?php /* Used by views/modal.js */?>
 		<script type="text/html" id="tmpl-brightcove-media-modal">
             <div class="media-modal wp-core-ui">
-	            <button type="button" class="media-modal-close"><span class="brightcove media-modal-icon"><span class="screen-reader-text"><?php esc_html_e( 'Close media panel', 'brightcove' ); ?></span></span></button>
+	            <button type="button" class="brightcove media-modal-close"><span class="brightcove media-modal-icon"><span class="screen-reader-text"><?php esc_html_e( 'Close media panel', 'brightcove' ); ?></span></span></button>
```

Comment:
Looks like the "brightcove" class on the icon isn't necessary, since it was only there as a js hook.